spec/secrets.yml

```diff
@@ -142,7 +142,7 @@ components:
         - "secrets"
         summary: "Fetch multiple secrets"
         description: "Fetches multiple secret values in one invocation. It’s faster to fetch secrets in batches than to fetch them one at a time."
-        operationId: "getVariables"
+        operationId: "showBatch"
```

Comment:
whatever we do above we should do similar here. the CLI just allows you to optionally input a list

we could call this `getMany` or `getMultiple` ... will people outside of API experts know what we mean by "batch"?

Reply:
I didn't really have an issue with `batch` when I first saw this endpoint. `batch` is also the [terminology](https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/Developer/Conjur_API_Batch_Retrieve.htm?tocpath=Developer%7CREST%C2%A0APIs%7C_____11) used in the documentation so I would prefer to stick to that if there is no existing standard. Will change to `getBatch`

Reply:
yeah, the internal methods are called `batch`, so `getBatch` it is

Reply:
or maybe `getBatchSecrets` and `getSecret`, if we are updating the methods to include the noun